cyberark/conjur
# An api key can be reset for an account

### Testing Instructions
1) modify the CPanel `dev/docker-compose.yml` entry for `possum_cpanel` as follows:
    - change `ports` value to `3000:3000`
1) modify the CPanel `dev/docker-compose.yml` entry for `possum` as follows:
    - change `image` to `conjur-dev` (run Conjur’s `build.sh` if you haven’t already)
    - add `entrypoint: conjurctl`
    - add `volumes` value `../../conjur:/src/conjur-server`
1) run `dev/start.sh`
1) follow Conjur README to set up conjur_docs container
1) visit eval page on localhost:4000
1) create a hosted Conjur account, take note of the email you used and the API key
1) use a cookie manager to delete your `accounts` cookie
1) refresh the page
1) sign up again w/ the same email address
1) observe the reset API key form appears
1) click reset
1) observe the new API key is displayed
